core/src/main/scala/org/http4s/AttributeMap.scala

```diff
@@ -85,6 +77,6 @@ object AttributeMap
 // type inference required less generality
 /** A map entry where `key` is constrained to only be associated with a fixed value of type `T`. */
 final case class AttributeEntry[T](key: AttributeKey[T], value: T) {
-  override def toString: String = key.name + ": " + value
+  override def toString: String = s"$key: $value"
```

Comment:
Yeah, I think this is a step forward, and if debugging gets annoying or someone comes along and says it's easy, we can do the Stupid Macro Trick that sbt's `project` uses.